qbft/controller.go

```diff
@@ -42,18 +42,26 @@ func NewController(
 }
 
 // StartNewInstance will start a new QBFT instance, if can't will return error
-func (c *Controller) StartNewInstance(value []byte) error {
-	if err := c.canStartInstanceForValue(value); err != nil {
-		return errors.Wrap(err, "can't start new QBFT instance")
+func (c *Controller) StartNewInstance(height Height, value []byte) error {
+	if err := c.GetConfig().GetValueCheckF()(value); err != nil {
+		return errors.Wrap(err, "value invalid")
+	}
+
+	if height < c.Height {
+		return errors.New("attempting to start an instace with a past or current height")
```

Comment:
the error says past or current but if `height == `c.Height` we will not fall into this codepath. should we change the error or the check to `=<`?

Reply:
This will be fixed when we merge the PR from Alon's repo.
The reason is that tests have an `ExpectedError` field and this change encompasses files not from this PR

Reply:
https://github.com/alonmuroch/ssv-spec/pull/4/commits/71d9fbe2cb8ba7cb4bd136a7885b20e812323334